pkg/packagekit/updates.jsx

```diff
@@ -172,9 +172,9 @@ function HeaderBar(props) {
             actionButton = <button className="btn btn-default" onClick={props.onRefresh} >{_("Check for Updates")}</button>;
         if (props.timeSinceRefresh !== null) {
             lastChecked = (
-                <span style={ {paddingRight: "3ex"} }>
```

Comment:
There shouldn't be inline CSS. This should be a CSS rule in the stylesheet. (I have it fixed in my mobile changes.)